web/angular.json

```diff
@@ -86,7 +91,6 @@
               "outputHashing": "all",
               "sourceMap": false,
               "namedChunks": false,
-              "aot": true,
```

Comment:
Just Curious, Is this because in angular 12 AOT is done by default?

Reply:
Yes, the migration tool changed that since `aot` is on by default now, see [here](https://blog.ninja-squad.com/2021/05/12/angular-cli-12.0/)

Reply:
ohh thanks!